app/templates/menu-browse.html

```diff
@@ -8,29 +8,29 @@
         <h1 class="title">{{ title }}</h1>
 
         {% for restaurant in restaurants %}
-		<a href="{{ restaurant.link }}">
-			<div class="restaurant-item">
+		<div class="restaurant-item">
+			<a href="{{ restaurant.link }}">
```

Comment:
I don't think HTML allows nested <a> tags, so there needs to be a way such that the "Add restaurant" button within the block will be a link to the our endpoints rather than a link to the zomato page. You could try playing around with making this into a form with the submit action being either going to the add endpoint or taking the user to the zomato page.

Reply:
That was there for a place holder until the backend for that part is written.

Reply:
So for now, i would prefer to leave it as is until i can play with working backend.

Reply:
The issue isn't that the button doesn't go to anything, it's that because the entire thing is wrapped in an <a> tag, we can't have the button ever go to anything in the future. I think it would be best to have the button just do nothing for now.